### DISCUSSION TITLE
Unable to create my space with an ENS on rinkeby

### DISCUSSION THREAD

**Discussion by HamzaJawadTroon** [snapshot-labs/snapshot#1440] [Q&A] [answered]
I have ENS name and when I try to create a new space, it tells me to register ENS domain. And I've also tried using following link
https://snapshot.org/#/ENSDOMAIN/settings
and setting record on ENS but coming back to snapshot and refreshing page is not loading form to create space.

**Comment by metrox-eth** [comment]
What's your ENS name ?

**Reply by HamzaJawadTroon** [reply]
It is testbetter.eth and the network is rinkeby.

**Comment by metrox-eth** [comment, marked as answer]
I see, we do not support testnets for ENS space creation, your ENS needs to be on mainnet, after having setup your space you will be able to use testnets in your space and strategies. This is a feature, not a bug. We wants organizations on snapshot to have a strong ownership of their spaces.

**Reply by NenadJovanovski** [reply]
Why just not make duration for testnets small, like 1 week. I wanted to test to snapshot but the 80$ gas fee is kinda putting me off.

**Reply by jambestwick** [reply]
why not support testnet

**Reply by manpreet2000** [reply]
Agree with @NenadJovanovski why do i need to setup ENS with mainnet for testing

**Reply by metrox-eth** [reply]
Just to clarify:
It's only the ENS that needs to be on mainnet, we support testnets of several chains to test strategies.
We can provide shared test spaces. Ask Laurent or Chaitu on discord.
